xds/src/test/java/io/grpc/xds/internal/sds/CommonTlsContextTestsUtil.java

```diff
@@ -170,7 +174,10 @@ static String getTempFileNameForResourcesFile(String resFile) throws IOException
    * Helper method to build DownstreamTlsContext for above tests. Called from other classes as well.
    */
   public static DownstreamTlsContext buildDownstreamTlsContextFromFilenames(
-      @Nullable String privateKey, @Nullable String certChain, @Nullable String trustCa) {
+      @Nullable String privateKey,
+      @Nullable String certChain,
+      @Nullable String trustCa,
+      boolean requireClientCert) {
```

Comment:
consider make a 2 methods sets `requirClientCert` and make this method private. so existing test doesn't need to be modified. although name is already long, this can be more descriptive.

Reply:
So you are suggesting 2 (public) methods like
```
buildDownstreamTlsContextFromFilenames
buildDownstreamTlsContextFromFilenamesRequireClientCert
```
calling this method with requireClientCert=false and requireClientCert=true respectively (and make this one private)?

Reply:
changed made.

Reply:
yes